update multi-device section (#632)
deltachat/deltachat-pages: en/help.md

### Can I use Delta Chat on multiple devices at the same time?

Yes. Delta Chat 1.36 comes with a new, experimental function for using the same account on different devices:

> - On the old device, go to **Settings → Add Second Device**, unlock the screen if needed
>   and wait a moment until a QR code is shown
> 
> - On the new device, install Delta Chat (<https://get.delta.chat>)
> 
> - Start Delta Chat on the new device, select **Add as Second Device** and scan the QR code from the old device

Reviewer: ```suggestion
- On the first device, go to **Settings → Add Second Device**, unlock the screen if needed
  and wait a moment until a QR code is shown

- On the second device, [install Delta Chat](https://get.delta.chat)

- On the second device, start Delta Chat, select **Add as Second Device**, and scan the QR code from the old device
```

To me old/new implies transferring data for single-device usage.

- Transfer should start after a few seconds and during transfer both devices will show the **progress**.
  Wait until it is finished on both devices.

In contrast to many other messengers, after successful transfer,
both **devices are completely independent.**

> The one device is not needed for the other to work.

Reviewer 2: maybe "the first device", "the one device" sounds weird to me, but maybe just because I am no native speaker

Reviewer 2: or "One device is not needed for another to work" maybe

Reviewer: ```suggestion
One device is not needed for the other to work.
```

> ### Troubleshooting

Reviewer 3: ```suggestion
#### Troubleshooting
```

This should be a sub-headline of "Can I use Delta Chat on multiple devices?"

Author: in theory, yes, maybe. but one can see this also for "troubleshooting for multidevice" - "enough storage", "already installed?" eg. also fits for "manual transfer".

in practise, we do not have 4th level headlines in the FAQ and it does not look good in the content (small screens). so, i  think, we should be too bureaucratic here, it is just the next section.

- Double-check both devices are in the **same Wi-Fi or network**

> - Your system may have a "personal firewall",

Reviewer 3: ```suggestion
- Your system might have a "personal firewall",
```
"may" has the connotation of "is allowed to" - in this context we mean the opposite.

>   this is known to make problems, esp. on Windows.

Reviewer 3: ```suggestion
  this is known to cause problems, especially on Windows.
```
slightly better wording.

> - Your system might have a "personal firewall",
>   this is known to cause problems, especially on Windows.

Reviewer: ```suggestion
- Your system might have a "personal firewall",
  which is known to cause problems (especially on Windows).
```

  **Disable the personal firewall** for Delta Chat on both ends and try again

- Ensure there is **enough storage** on the destination device

- If transfer started, make sure, the devices **stay active** and do not fall asleep.
  Do not exit Delta Chat.

>   (we try hard to work in background, but systems tend to kill apps, unfortunately)

Reviewer 3: ```suggestion
  (we try hard to make the app work in background, but [systems tend to kill apps](https://dontkillmyapp.com), unfortunately)
```

> - Delta Chat is **already installed** on the destination device?

Reviewer 3: ```suggestion
- Delta Chat is **already logged in** on the destination device?
```
Let's keep it precise. People who installed the app according to our guide, but didn't login yet, could think this point applies to them.

I admit that my suggestion has a slightly weird wording. But technically, it *is* Delta Chat that is logged in to the email account :sweat_smile: maybe someone else can make up something better.

  You can use multiple accounts per device, just [add another account](#multiple-accounts)

- If you still have problems or if you **cannot scan a QR code**
  try the **manual transfer** described below

> ### Manual Transfer

Reviewer 3: ```suggestion
#### Manual Transfer
```
Should be a sub-headline as well.
